packages/build/package.json

```diff
@@ -40,10 +40,11 @@
     "prettier:check": "npm run prettier:cli -- -l",
     "prettier:fix": "npm run prettier:cli -- --write",
     "test": "npm run mocha",
-    "mocha": "node bin/select-dist mocha --timeout 30000 --opts mocha.opts \"test/integration/*.js\"",
+    "mocha": "node bin/select-dist mocha --timeout 30000 \"test/integration/*.js\"",
     "posttest": "npm run lint"
   },
   "devDependencies": {
-    "fs-extra": "^5.0.0"
+    "fs-extra": "^5.0.0",
+    "source-map-support": "^0.5.2"
```

Comment:
Since `source-map-support` is TypeScript specific, there's no need for it in `build` right?

Reply:
Well `build` is using the same shared `mocha.opts` that TypeScript projects are using.

If we don't want to have `source-map-support` as part of build's mocha opts, then we would have to create (and maintain) another copy, one without `--require source-map-support`.

Not a big deal, I don't really mind either way.